src/Symfony/Component/Security/Core/User/UserInterface.php

```diff
@@ -55,7 +55,7 @@ public function getRoles(): array;
     public function eraseCredentials();
 
     /**
-     * Returns the identifier for this user (e.g. its username or email address).
+     * Returns the identifier for this user (e.g. username or email address).
```

Comment:
their?

Reply:
If you want, I can surely add "their". But IMO it's clear anyway in this context about *whose* username we're talking...